examples/05-Retrieval-Model.ipynb

```diff
@@ -61,7 +61,8 @@
     "### Learning objectives\n",
```

Comment:
<p>the <code>to_top_k_encoder()</code> is a method of the RetrievalModelV2 class that uses the pre-trained candidate and query encoders to initialize a top-k encoder model,  called the<code>topk_model</code> in this example.  Practically, this method applies the <code>candidate_encoder</code> on the provided <code>candidate_features</code> dataset to set the top-k index of the <code>topk_model</code>  .</p><p> </p><p>so the <code>topk_model</code> object is the one responsible of generating the top-k predictions. In fact, the forward method of <code>topk_model</code> takes as input the query features and computes the dot product scores between the given query embeddings and all the candidates of the top-k index. Then, it returns the top-k (k=20) item ids with the highest scores. </p><p> </p><p>The output of the method is a named tuple <code>TopKPrediction</code> , where the first element is the dot product scores and the second element is the encoded item ids. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/900/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='69'/>